config/services.yml

```diff
@@ -666,6 +666,32 @@ https://cmr.uat.earthdata.nasa.gov:
       - image: !Env ${QUERY_CMR_IMAGE}
       - image: !Env ${SWOT_REPROJECT_IMAGE}
 
+  - name: sds/harmony-regridder
+    # The Harmony Regridding Service for L3/L4 collections
+    data_operation_version: '0.17.0'
+    type:
+      <<: *default-turbo-config
+      params:
+        <<: *default-turbo-params
+        env:
+          <<: *default-turbo-env
+          STAGING_PATH: public/sds/harmony-regridder
+    umm_s:
+      - S1256563664-EEDTEST
```

Comment:
I couldn't find this service in the UAT MMT. Should I be able to?

Reply:
Discussed in Slack:

Matt used the service concept ID in the search bar in MMT. This did not retrieve the UMM-S record. (Nor for me)
Searching MMT for the service name "Harmony Regridding Service" was successful, though.

[Here is the URL for the service in UAT CMR](https://mmt.uat.earthdata.nasa.gov/services/S1256563664-EEDTEST). Alternatively, [here is the URL for the CMR search response for this UMM-S record](https://cmr.uat.earthdata.nasa.gov/search/services.umm_json?concept_id=S1256563664-EEDTEST).